docs/sql-data-sources-parquet.md

```diff
@@ -432,4 +498,32 @@ Configuration of Parquet can be done using the `setConf` method on `SparkSession
   </td>
   <td>3.1.0</td>
 </tr>
+<tr>
+  <td>parquet.encryption.column.keys</td>
+  <td><code>none</code></td>
+  <td>List of columns to encrypt, with master key IDs. In the form of <code>masterKeyID:colName,colName;masterKeyID:colName...</code>
+  </td>
+  <td>3.2.0</td>
+</tr>
+<tr>
+  <td>parquet.encryption.double.wrapping</td>
+  <td><code>true</code></td>
+  <td>By default the data encryption keys (DEKs) are encrypted with key encryption keys (KEKs), which in turn are encrypted with master keys. If set to false, DEKs are directly encrypted with master keys, KEKs are not used.
+  </td>
+  <td>3.2.0</td>
+</tr>
+<tr>
+  <td>parquet.encryption.key.material.store.internally</td>
+  <td><code>true</code></td>
+  <td>By default, the key material (encrypted DEKs) are stored inside Parquet file footers - this doesn’t produce additional files. If set to false, key material is stored in separate files in the same folder - this enables master key rotation for immutable Parquet files.
+  </td>
+  <td>3.2.0</td>
+</tr>
+<tr>
+  <td>parquet.encryption.data.key.length.bits</td>
+  <td><code>128</code></td>
+  <td>Length of data encryption keys (DEKs), randomly generated by Parquet key management tools. Can be 128, 192 or 256 bits.
+  </td>
+  <td>3.2.0</td>
+</tr>
```

Comment:
Could you remove this addition? As @sunchao and you discussed, we don't duplicate like this. Instead, you can mention them in your examples and give a pointer to Apache Parquet page.

Reply:
Thats a good idea, we have a nice page with these parameters. Will do.